src/box/txn_limbo.c

```diff
@@ -802,13 +802,12 @@ txn_limbo_wait_last_txn(struct txn_limbo *limbo, bool *is_rollback,
 }
 
 int
-txn_limbo_wait_confirm(struct txn_limbo *limbo)
+txn_limbo_wait_confirm(struct txn_limbo *limbo, double timeout)
```

Comment:
Why do you need to pass `timeout`, when you can always use `replication_wait_confirm_timeout()` like it was done before?

Reply:
Fixed.